Neos.Neos/Classes/FrontendRouting/Projection/DocumentUriPathProjection.php

```diff
@@ -208,13 +209,15 @@ private function whenRootNodeAggregateDimensionsWereUpdated(RootNodeAggregateDim
             return;
         }
 
+        $oneCoveredDimensionSpacePoint = current(iterator_to_array($event->coveredDimensionSpacePoints));
+        if (!$oneCoveredDimensionSpacePoint instanceof DimensionSpacePoint) {
+            return;
+        }
+
         // Just to figure out current NodeTypeName. This is the same for the aggregate in all dimensionSpacePoints.
-        $pointHashes = $event->coveredDimensionSpacePoints->getPointHashes();
-        $anyPointHash = reset($pointHashes);
-        assert(is_string($anyPointHash));
```

Comment:
just out of curiosity @nezaniel - is there actually a guarantee that in this case at least one dimension is covered?

I dont understand 😅 ... because $coveredDimensionSpacePoints also has an `isEmpty` method ...

Reply:
I'm pretty sure the command handlers catch commands that wouldn't apply to anything at all ^^

Reply:
... i still cant wrap my head around what the return value would be for a neos cr without dimensions (0 dimensional)? Isnt empty an expected case?

Reply:
no, an empty dimension space point is still a dimension space point

Reply:
Hmm a i see. That makes sense. But why does the `DimensionSpacePointSet` to be empty.